scipy/special/add_newdocs.py

```diff
@@ -6369,3 +6369,36 @@ def add_newdoc(place, name, doc):
     """
     Internal function, do not use.
     """)
+
+add_newdoc("scipy.special", "owens_t",
+    """
```

Comment:
Begin the documentation with a one sentence definition; i.e. "Owen's T Function."

Then there's a blank line, followed by the main definition.